packages/react-native-renderer/src/ReactFiberConfigFabric.js

```diff
@@ -228,12 +231,14 @@ export function getChildHostContext(
   type: string,
 ): HostContext {
   const prevIsInAParentText = parentHostContext.isInAParentText;
-  const isInAParentText =
-    type === 'AndroidTextInput' || // Android
-    type === 'RCTMultilineTextInputView' || // iOS
-    type === 'RCTSinglelineTextInputView' || // iOS
-    type === 'RCTText' ||
-    type === 'RCTVirtualText';
+  const isInAParentText = enableViewConfigParentTextContext
+    ? getViewConfigForType(type).supportsRawText
```

Comment:
Probably want to coalesce this to `false` if it's not normalised elsewhere.

```suggestion
    ? getViewConfigForType(type).supportsRawText ?? false
```